django/core/handlers/base.py

```diff
@@ -246,6 +247,11 @@ def get_response(self, request):
 
         response._closable_objects.append(request)
 
+        # If the exception handler is a CBV, then force template to render.
```

Comment:
"If the exception handler returns a TemplateResponse, that has not been rendered so far, force a rendering"